lib/ansible/modules/system/service_facts.py

```diff
@@ -184,13 +184,13 @@ class SystemctlScanService(BaseService):
     def systemd_enabled(self):
         # Check if init is the systemd command, using comm as cmdline could be symlink
         try:
-            f = open('/proc/1/comm', 'r')
+            with open('/proc/1/comm', 'r') as f:
```

Comment:
Same here: you now not only ignore errors during opening, but also during reading. This is potentially changing semantics!